.github/workflows/update-tags.yml

```
@@ -23,6 +23,28 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           force: false
+  update-gardenctl-v2:
+    if: github.event.client_payload.component == 'gardenctl-v2'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Event Information
+        run: |
+          echo "Event '${{ github.event.action }}' payload '${{ github.event.client_payload.tag }}'"
+          echo "Mac gardenctl-v2 binary hash '${{ github.event.client_payload.mac_sha }}'"
+          echo "Linux gardenctl-v2 binary hash '${{ github.event.client_payload.linux_sha }}'"
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # pin@v2.4.0
+      - name: Run update script
+        run:  ./.github/workflows/update-gardenctl-v2.sh ${{ github.event.client_payload.tag }} ${{ github.event.client_payload.darwin_sha }} ${{ github.event.client_payload.linux_sha }}
+      - name: Commit files
+        run: |
+          git config --local user.email "gardener.ci.user@gmail.com"
+          git config --local user.name "gardener-robot-ci-1"
+          git commit -m "Update gardenctl-v2" -a
```

```suggestion
          git add gardenctl-v2.rb
          git commit -m "Update gardenctl-v2" -a
```
then you do not need to commit an empty `gardenctl-v2.rb` file